crates/gitbutler-branch-actions/src/actions.rs

```diff
@@ -79,7 +79,7 @@ pub fn list_virtual_branches(project: &Project) -> Result<StackListResult> {
 
 pub fn list_virtual_branches_cached(
     project: &Project,
-    worktree_changes: Option<DiffByPathMap>,
+    diffs_by_path: DiffByPathMap,
```

Comment:
The original name was more descriptive - the new name just repeats the type name